scripts/hail_batch/gnomad_90k_variants_liftover/README.md

```diff
@@ -0,0 +1,9 @@
+# Perform liftover on the gnomad v2 loadings
+
+This runs a Hail query script in Dataproc using Hail Batch in order to perform liftover on the gnomAD v2 loadings (GRCh37 -> GRCh38). To run, use conda to install the analysis-runner, then execute the following command:
+
+```sh
+analysis-runner --dataset ancestry \
+--access-level test --output-dir "gs://cpg-reference-temporary/gnomad_loadings_liftover/v0" \
```

Comment:
Did you mean `cpg-ancestry-temporary`?

(There's a `cpg-reference` bucket in the `cpg-common` GCP project. Probably easiest to copy any results (ideally with some sort of description) there using your personal credentials.)

Might be good to add a gnomAD version in here as well (e.g. 2.1), just in case they decide to release newer ones at some point.